File: ytrue-admin-server-csharp/Ytrue.Infrastructures/Response/IServerResponseInfo.cs
namespace Ytrue.Infrastructures.Response;

public interface IServerResponseInfo
{
    /// <summary>
    /// 获取code
    /// </summary>
    /// <returns></returns>
    string Code();
    
    
    /// <summary>
    /// 获取错误信息
    /// </summary>
    /// <returns></returns>
    string Message();
}

File: ytrue-admin-server-java/ytrue-infra/ytrue-infra-core/src/main/java/com/ytrue/infra/core/base/IServerResponseInfo.java
package com.ytrue.infra.core.base;

/**
 *
 * @author ytrue
 * @date 2024/10/19
 * @description  服务器响应信息接口，定义了响应的状态码和消息。
 *
 * <p>
 * 该接口用于统一响应格式，方便在应用中处理不同的响应信息。
 * </p>
 */
public interface IServerResponseInfo {

    /**
     * 获取状态码。
     *
     * @return 返回响应的状态码
     */
    String code();

    /**
     * 获取响应消息。
     *
     * @return 返回响应的消息内容
     */
    String message();

}
